src/app/storage.ts

```diff
@@ -51,29 +46,24 @@ export function performMigrations(runningInBrowser: boolean, storageInstance: St
  */
 function getOldFavorites(runningInBrowser: boolean): void {
   // Only do this specific migration if we're a native app.
-  console.log('PVTrAck database migration 1');
   // Regardless of device type, we're no longer worried about
   // whether the user is coming from 1.x
   localforage.removeItem('returningUser');
   if (!runningInBrowser) {
     storage.ready().then(() => {
-      console.log(`Running on native device with driver ${storage.driver}`);
       // Only do this migration if this device is capable of using
       // native SQLite storage
       // (ionic falls back to localforage if not, in which case we're done)
       if (storage.driver === 'sqlite'
       || storage.driver === 'cordovaSQLiteDriver') {
         // Move each thing from PVTrAck 1.x's storage to 2.0's.
-        console.log('Copying PVTrAck 1.x\'s storage');
         localforage.iterate((value, key, iterationNumber) => {
-          console.log([key, value]);
           storage.set(key, value);
         }).then(() => {
           // In success, we no longer need 1.x's storage.
-          console.log('PVTrAck 1.x storage copy completed');
           localforage.clear();
         }).catch((err) => {
-          console.log(err);
+          console.error(err);
```

Comment:
This happens in other places, but I'm not sure what to do with it since we're catching the error.